templates/patterns/modal/modal-language.html.twig

```diff
@@ -12,7 +12,8 @@
     - footer
       - label
 #}
-
+{% set eu_official_languages = 'EU official languages' %}
```

Comment:
|t for both, see if we can get the translations from oe_theme or we need to ask for them